modules/tracking/src/trackerKCF.cpp

```diff
@@ -47,7 +47,7 @@
 |---------------------------*/
 namespace cv{
    /**
-  * \brief Implementation of TrackerModel for MIL algorithm
+  * \brief Implementation of TrackerModel for KCF algorithm
   */
   class TrackerKCFModel : public TrackerModel{
```

Comment:
Similar problem is [here](https://github.com/opencv/opencv_contrib/blob/3.3.0/modules/tracking/doc/%5BTutorial%5D%20Adding%20new%20Tracker%20Method%20for%20dummies) too